<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/img/icon_light/settings.svg</normaloff>:/img/icon_light/settings.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_settings">
     <property name="styleSheet">
      <string notr="true">
QFrame#frame_settings{
	background-color: #1b1b23;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{border:none}

QLineEdit {
    color: #f2f2f2;
	border-bottom: 1px solid #0580d9;
    background-color: transparent; 
}

QLineEdit:focus {
    border-bottom: 1px solid #f2f2f2 ; 
}

QLabel {
    background-color: transparent;
    font: 75 10pt 'Arial ';
    font-weight: Normal;
}

QCheckBox::indicator:unchecked {
    image: url(&quot;:/img/icon_light/toggle-button-left.png&quot;);
    width: 40px; 
    height: 40px; 
}

QCheckBox::indicator:checked {
    image: url(&quot;:/img/icon_light/toggle-button-right.png&quot;);
    width: 40px; 
    height: 40px; 
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Editar tabla</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="chxb_edit_tb_mode">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ocultar barra estado</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="le_vacantes">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Precio Matricula</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="chxb_hide_DB_status_bar">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="chexb_auto_refresh_tb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator:unchecked {
    image: url(&quot;:/img/icon_light/toggle-button-left.png&quot;);
    width: 40px; 
    height: 40px; 
}

QCheckBox::indicator:checked {
    image: url(&quot;:/img/icon_light/toggle-button-right.png&quot;);
    width: 40px; 
    height: 40px; 
}


</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="le_precio">
         <property name="maximumSize">
          <size>
           <width>34</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Autore refrescar </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Vacantes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="btx_settings">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(34, 34, 44);</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
